package.json

```diff
@@ -41,6 +47,9 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.14.2",
     "jest": "^24.8.0",
+    "mongodb": "^3.3.0-beta2",
```

Comment:
`mongodb` and `mongoose` are not devDependencies.

Reply:
moved them to dependencies, thank you Will!